sources/packages/backend/libs/test-utils/src/factories/student-file-uploads.ts

```diff
@@ -0,0 +1,41 @@
+import * as faker from "faker";
+import { FileOriginType, Student, StudentFile, User } from "@sims/sims-db";
+import { DataSource } from "typeorm";
+import { createFakeStudent } from "./student";
+
+/**
+ * Create fake student file upload object.
+ * @param student entity.
+ * @returns created studentFile object.
+ */
+export function createFakeStudentFileUpload(student?: Student): StudentFile {
+  const studentFile = new StudentFile();
+  studentFile.fileName = faker.system.fileName();
+  studentFile.uniqueFileName =
+    studentFile.fileName +
+    faker.random.number(5) +
```

Comment:
Even the DB not having a constraint to the `unique_file_name` the file names are generated including a guid to ensure their uniqueness, for instance, `Upload Test - Copy-2c0028d5-c681-46d4-84b3-471d890a6f13.doc`.
Can we please ensure that this factory does the same?

Reply:
Are you suggesting that I use a `guid` generator instead of `faker.random.number(5)` that is currently in the code?